### PR TITLE
Add info section about phpList hosted to Automatic Updater screens

### PR DESCRIPTION
**Pull request by BrunildaDosti**
Mantis link:
https://mantis.phplist.org/view.php?id=19744

### CODE REVIEW

**Comment by suelaP** [comment]
The changes have introduced functional changes.
1. The "Next" button seems to be broken 
2. The "Updater is loading.There is a new update available!" texts is hardcoded and it will show both when there is an update available and when not.
3. On Google Chrome the "Next" button (using the default display settings) will not show on the screen but needs scrolling, while the arrow section is displayed.
4. On Mozilla Firefox the arrow animation won't work.

**Comment by samtuke** [comment]
@suelaP Were you testing with or without my changes?

**Comment by suelaP** [comment]
All these were there before your changes but I tested again with your changes and no changes on any of these points.

**Comment by samtuke** [comment]
@suelaP Makes sense; my changes were only to the display text

File: index.php [hunk 1]
```diff
@@ -1034,190 +1091,644 @@ function replaceNewUpdater() {
...
                 <h2>Perform update</h2>
             </div>
             <div class="clear"></div>
         </div>
+
         <div id="display">
             <span id="current-step" class="hidden"> </span>
-            <span id="success-message">Updater is loading.</span><br>
+            <span id="success-message">Updater is loading.<br/>
+             There is a new update available!</span><br>
```
Comment (xh3n1) [inline index.php]: @BrunildaDosti Please do not hardcode this.

**Comment by samtuke** [comment]
Each of the tick icons next to the bullet points seems to be identical but duplicated (svg). Please try and avoid the duplication.

**Comment by xh3n1** [comment]
There is a merge conflict that should be resolved.

**Comment by xh3n1** [comment]
I would suggest using SVGs for the images/icons.

**Comment by samtuke** [comment]
Regarding the merge conflict: @xh3n1 @BrunildaDosti you need to negotiate between you who will resolve it and go ahead and do it. Better communication can avoid such situations.

**Comment by xh3n1** [comment]
@BrunildaDosti I will resolve the conflict if you don't mind.

**Comment by BrunildaDosti** [comment]
> @BrunildaDosti I will resolve the conflict if you don't mind.

Thanks @xh3n1 . Please go ahead.

**Comment by xh3n1** [comment]
Resolved conflicts, added "go back to dashboard link" and pointer cursor to logo, tested and now merging.

**Pull request merged**